bin/db_patcher.js

```diff
@@ -23,8 +23,8 @@ options.reversePatchAllowed = false
 
 patcher.patch(options, function(err) {
   if (err) {
-    log.error(err)
+    logger.error('patch-error', { err : '' + err })
     process.exit(2)
```

Comment:
If I just pass the `err` here as the 2nd argument, then the output just shows `"Fields":{}`. I'd somewhat expected the stringified message to appear somehow but it didn't, so I made it into an object with the error string.

This may not matter here much since `db_patcher.js` is only used in dev, but I do wonder if there is a better way.